docs/source/release_notes.rst

```diff
@@ -134,12 +254,10 @@ Bugfixes
 - ``zos_find`` - Option size failed if a PDS/E matched the pattern, now filtering on utilized size for a PDS/E is supported.
 - ``zos_mvs_raw`` - Option **tmp_hlq** when creating temporary data sets was previously ignored, now the option honors the High Level Qualifier for temporary data sets created during the module execution.
 
-Availability
-------------
+- ``zos_tso_command``
 
-* `Automation Hub`_
-* `Galaxy`_
-* `GitHub`_
+    - Has been updated with a new example demonstrating how to explicitly execute a REXX script in a data set.
+    - Has been updated with a new example demonstrating how to chain multiple TSO commands into one invocation using semicolons.
```

Comment:
the above 2 lines may be another potential merge error/issue

Reply:
Fixed the release notes